src/features/unified-share-modal/SharedLinkPermissionMenu.js

```diff
@@ -77,7 +77,14 @@ class SharedLinkPermissionMenu extends Component<Props> {
         };
 
         return (
-            <DropdownMenu constrainToWindow>
+            <DropdownMenu
+                constrainToWindow
+                onMenuOpen={() => {
+                    if (canShow && sharedLinkEditTagTargetingApi) {
```

Comment:
sharedLinkEditTagTargetingApi is guaranteed to exist if canShow is true
```suggestion
                    if (canShow) {
```

Reply:
I added this second part because there was a flow issue indicating that sharedLinkEditTagTargetingApi could have been undefined.

Reply:
I do agree with the above statement though - it seems like flow isn't able to infer that properly.

Reply:
Yeah I can use that value